src/elements/profiles/Profiles.wc.svelte

```diff
@@ -300,17 +300,22 @@
     <div class="box">
       <div class="is-flex is-justify-content-space-between">
         <h4 class="is-size-4">Profile Manager</h4>
-        <button
-          class="button"
-          class:is-link={migrateMode}
-          class:is-loading={migrating}
-          disabled={migrating || !mnemonics$.valid}
-          on:click={() => (migrateMode = !migrateMode)}
-          style:background-color="#1982b1"
-          style:color="white"
-        >
-          Migrate
-        </button>
+        <div class="d-flex flex-column">
+          <button
+            class="button"
+            class:is-link={migrateMode}
+            class:is-loading={migrating}
+            disabled={migrating || !mnemonics$.valid}
+            on:click={() => (migrateMode = !migrateMode)}
+            style:background-color="#1982b1"
+            style:color="white"
+          >
+            Migrate
+          </button>
+          <button class="button" style:background-color="#e0e0e0" style:color="black" on:click={setShow(false)}>
```

Comment:
What setShow mean? if you using this function to open and close the profile modal you can change its name with a clean name, e.g
```suggestion
          <button class="button" style:background-color="#e0e0e0" style:color="black" on:click={profileToggle(false)}>
```

also instead of using true and false to enable and disable the profile modal, you can use the js `toggle` class to do it. 
https://www.w3schools.com/howto/howto_js_toggle_hide_show.asp

or use active class
 
https://svelte.dev/repl/e85cdaba377341c181134646c6df69ce?version=3.18.2

Reply:
`setshow` function takes the value of its pram and set `class:is-active` with show value,

Reply:
@0oM4R Please update the function name as from `setShow` to `profileToggle`